openstudiocore/src/gbxml/ForwardTranslator.cpp

```diff
@@ -1171,6 +1343,52 @@ namespace gbxml {
     result.appendChild(nameElement);
     nameElement.appendChild(doc.createTextNode(QString::fromStdString(name)));
 
+    // heating setpoint
+    boost::optional<double> designHeatT;
+    boost::optional<double> designCoolT;
+    boost::optional<model::Thermostat> thermostat = thermalZone.thermostat();
+    if (thermostat && thermostat->optionalCast<model::ThermostatSetpointDualSetpoint>()){
```

Comment:
Would be great to have a method like Thermostat::coolingDesignSetpoint and Thermostat::heatingDesignSetpoint